### PR TITLE
Update perf tool

### PR DESCRIPTION
**Pull request by dangleptr**

### CODE REVIEW

File: src/tools/storage-perf/StoragePerfTool.cpp [hunk 1]
```diff
@@ -177,7 +186,7 @@ class Perf {
 
     void getVerticesTask() {
         auto* evb = threadPool_->getEventBase();
-        auto f = client_->getVertexProps(defaultSpaceId_, randomVertices(), randomCols())
+        auto f = client_->getVertexProps(FLAGS_default_space_id, randomVertices(), randomCols())
```
Comment (whitewum) [inline src/tools/storage-perf/StoragePerfTool.cpp]: is random() fast enough?
any benchmark?

File: src/tools/storage-perf/StoragePerfTool.cpp [hunk 2]
```diff
@@ -34,13 +35,21 @@ class Perf {
         LOG(INFO) << "Total threads " << FLAGS_threads
                   << ", qpsPerThread " << qpsPerThread
                   << ", task interval ms " << interval;
+        auto metaAddrsRet = nebula::network::NetworkUtils::toHosts(FLAGS_meta_server_addrs);
```
Comment (whitewum) [inline src/tools/storage-perf/StoragePerfTool.cpp]: not hard code 1000 in many lines.

File: src/tools/storage-perf/StoragePerfTool.cpp [hunk 3]
```diff
@@ -34,13 +35,21 @@ class Perf {
...
         std::vector<std::unique_ptr<thread::GenericWorker>> threads;
         for (int32_t i = 0; i < FLAGS_threads; i++) {
             auto t = std::make_unique<thread::GenericWorker>();
             threads.emplace_back(std::move(t));
         }
         threadPool_ = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_io_threads);
-        client_ = std::make_unique<StorageClient>(threadPool_, nullptr);
+        mClient_ = std::make_unique<meta::MetaClient>(threadPool_, metaAddrsRet.value());
+        CHECK(mClient_->waitForMetadReady());
+        client_ = std::make_unique<StorageClient>(threadPool_, mClient_.get());
```
Comment (whitewum) [inline src/tools/storage-perf/StoragePerfTool.cpp]: line 69. Due to multiple threading, is it possible the main thread skip 1000*X after some sleep? e.g. 998->sleep->1002

File: src/tools/storage-perf/StoragePerfTool.cpp [hunk 4]
```diff
@@ -13,14 +13,15 @@
 DEFINE_int32(threads, 2, "Total threads for perf");
 DEFINE_int32(qps, 1000, "Total qps for the perf tool");
 DEFINE_int32(totalReqs, 10000, "Total requests during this perf test");
-DEFINE_int32(io_threads, 3, "Client io threads");
-DEFINE_bool(mock_server, true, "Test server was started in mock server mode");
-DEFINE_int32(mock_server_port, 44500, "The mock server port");
+DEFINE_int32(io_threads, 10, "Client io threads");
```
Comment (whitewum) [inline src/tools/storage-perf/StoragePerfTool.cpp]: more totalRes, 100000

File: src/tools/storage-perf/StoragePerfTool.cpp (hunk 1 above)
Comment (dangleptr) [reply src/tools/storage-perf/StoragePerfTool.cpp]: Looks good now.

File: src/tools/storage-perf/StoragePerfTool.cpp (hunk 2 above)
Comment (dangleptr) [reply src/tools/storage-perf/StoragePerfTool.cpp]: Yes,  it will be changed in #721

**Comment by dangleptr** [comment]
Jenkins go

**Comment by dangleptr** [comment]
Jenkins go

**Comment by dangleptr** [comment]
Jenkins go

**Pull request merged**